src/remote-desktop.c

```diff
@@ -539,7 +639,7 @@ start_done (GObject *source_object,
     {
       if (response == 0)
         {
-          if (!process_results (remote_desktop_session, results, &error))
+          if (!process_results (remote_desktop_session, &results, &error))
             {
               g_warning ("Could not start remote desktop session: %s",
                          error->message);
```

Comment:
Github doesn't let me comment on the correct line, so I'll add a reminder here instead.

`xdp_dbus_remote_desktop_set_version (XDP_DBUS_REMOTE_DESKTOP (remote_desktop), 1);`
needs to be changed to
`xdp_dbus_remote_desktop_set_version (XDP_DBUS_REMOTE_DESKTOP (remote_desktop), 2);`

Reply:
unrelated, but I've been using a `VERSION_V(1)` macro in [libei](https://gitlab.freedesktop.org/libinput/libei/-/blob/main/src/util-version.h) and it really makes these lines a lot easier to find and replace...

Reply:
Sounds useful - could add that for all instances of version checking/setting (as a follow up, I humbly suggest)

Reply:
Let's do it as a separate PR please

Reply:
Tracking this on https://github.com/flatpak/xdg-desktop-portal/issues/1049 now.